docs/services/data/configure-cloud-sync.md

```diff
@@ -72,7 +72,7 @@ In the example pictured here, the Data Management Service syncs the configured c
       "type": "data_manager",
       "attributes": {
         "sync_interval_mins": 5,
-        "capture_dir": "/.viam/capture",
+        "capture_dir": "~/.viam/capture",
```

Comment:
Can the capture_dir in this file be updated too?

Reply:
```suggestion
        "capture_dir": "",
```